src/main/java/com/capstone/producer/ServiceHandler.java

```diff
@@ -107,7 +127,7 @@ public ServiceHandler() {
      * or an error response.
      */
     public String handleFlightIdent(String flightIdent) {
-        LOGGER.debug("Received request: {}", flightIdent);
+        LOGGER.debug(RECEIVED_REQUEST_MSG, flightIdent);
```

Comment:
I find this harder to read since you can't simply see the string it's logging. An alternative would be to create methods specifically dedicated to processing those logs. 

Should be good enough for now though.